e2e/pages/portal/submission-success-page.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, type Locator, type Page } from '@playwright/test';
+
+export enum ApplicationStatus {
+  UnderReviewByALC = 'Under Review by ALC',
+  ReceivedByALC = 'Received By ALC',
+  SubmittedToALCIncomplete = 'Submitted to ALC - Incomplete',
+  SubmittedToALC = 'Submitted to ALC',
+  UnderReviewByLFNG = 'Under Review by L/FNG',
+  SubmittedToLFNG = 'Submitted to L/FNG',
+  InProgress = 'In Progress',
+  LFNGReturnedAsIncomplete = 'L/FNG Returned as Incomplete',
+  WrongLFNG = 'Wrong L/FNG',
+  ALCReturnedToLFNG = 'ALC Returned to L/FNG',
+  Cancelled = 'Cancelled',
+  LFNGRefusedToForward = 'L/FNG Refused to Forward',
+  DecisionReleased = 'Decision Released',
+}
+
+export class SubmissionSuccessPage {
+  readonly page: Page;
+  readonly heading: Locator;
+  readonly viewSubmissionButton: Locator;
+  readonly applicationStatusText: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.heading = page.getByRole('heading', { name: 'Application ID' });
+    this.viewSubmissionButton = page.getByRole('button', { name: 'view submission' });
+    this.applicationStatusText = page.getByTestId('application-status');
+  }
+
+  async fileId(): Promise<string> {
+    const headingText = await this.heading.textContent();
+
+    await expect(headingText).not.toBeNull();
+
+    const matches = headingText!.match(/(?<=Application ID: )\d*/);
+
+    await expect(matches).not.toBeNull();
+
+    const fileId: string = matches![0];
+
+    // Should never return empty string, but necessary to appease TypeScript
```

Comment:
Is this comment meant for the line above?

Reply:
I don't think this comment is even relevant anymore. I must have changed the way I did this and forgot to update comment. Good catch!